fetch.bs

```diff
@@ -2213,6 +2283,89 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<p>The requirements for <dfn>recording connection timing info</dfn> given a <a for=/>connection</a>
+<var>connection</var> and its <a for=/>connection timing info</a> <var>timingInfo</var>, are as
+follows:
+<ul>
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">domain lookup start time</a>
+ should be the <a for=/>unsafe shared current time</a> immediately before starting the domain
+ lookup, or beginning retrieval of the information from cache.
+
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">domain lookup end time</a> should
+ be the <a for=/>unsafe shared current time</a> immediately after finishing the domain lookup, or
+ retrieving the information from cache.
+
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">connection start time</a> should
+ be the <a for=/>unsafe shared current time</a> immediately before establishing the connection to
+ the server or proxy.
+
+ <li><p><var>timingInfo</var>'s <a for="connection timing info">connection end time</a> should be
+ the <a for=/>unsafe shared current time</a> immediately after establishing the connection to the
+ server or proxy, as follows:
+ <ul>
+  <li><p>The returned time must include the time interval to establish the transport
+  connection, as well as other time intervals such as SOCKS authentication. It
+  must include the time interval to complete enough of the TLS handshake to
+  request the resource.
+
+  <li><p>If the user agent used TLS False Start [[RFC7918]] for this connection,
+  this interval must not include the time needed to receive the server's
+  Finished message.
+
+  <li><p>If the user agent sends the request with early data [[RFC8470]] without
+  waiting for the full handshare to complete, this interval must not include
+  the time needed to receive the server's ServerHello message.
+
+  <li><p>If the user agent waits for full handshake completion to send the
+  request, this interval includes the full TLS handshake even if other
+  requests were sent using early data on <var>connection</var>.
+ </ul>
```

Comment:
Similar to above, we need to state what happens when a previous connection was reused

Reply:
See comment above.